devices/philips.js

```diff
@@ -1220,6 +1220,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['929003055601'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
```

Comment:
Drop the comment on this line please.

Reply:
Done. I kept the original commit message though I'm not sure that's optimal. Again, I'm pretty new to this part of git.

Reply:
Koen squashes the commits when merging so it's not a problem.